pyproject.toml

```diff
@@ -127,26 +127,12 @@ text_file_format = "rst"
 addopts = "--doctest-rst --import-mode=append"
 filterwarnings = [
     "error",
-    "ignore:numpy\\.ufunc size changed:RuntimeWarning",
```

Comment:
Need to keep the ufunc one. It comes up once in a while depending on how CI installed stuff and in what order.